mscore/palette.cpp

```diff
@@ -711,6 +714,29 @@ void PaletteScrollArea::keyPressEvent(QKeyEvent* event)
 
             p->update();
             }
+      else if ((event->key() == Qt::Key_Down || event->key() == Qt::Key_Up)
+               && !newScoreWizard) {
+            p->setCurrentIdx(-1);
+            if (p->name() == "Time Signatures")
```

Comment:
Will this work when the UI is transtated? I don't really like having reference to the master palette in palette.cpp, it should not be linked.